pkg/fleet/installer/packages/datadog_agent_windows.go

```diff
@@ -306,14 +337,23 @@ func removeAgentIfInstalled(ctx context.Context) (err error) {
 			}
 			span.Finish(err)
 		}()
-		err := msi.RemoveProduct("Datadog Agent")
+		err := msi.RemoveProduct(product)
 		if err != nil {
 			return err
 		}
 	} else {
-		log.Debugf("Agent not installed")
+		log.Debugf("%s not installed", product)
 	}
 	return nil
+
+}
+
+func removeAgentIfInstalled(ctx context.Context) (err error) {
+	return removeProductIfInstalled(ctx, "Datadog Agent")
+}
+
+func removeInstallerIfInstalled(ctx context.Context) (err error) {
```

Comment:
do we have to remove the package from disk and/or packages.db, too? so that it isn't reported to the backend.
Maybe we can run `datadog-installer.exe remove` ?